pkg/asset/installconfig/aws/regions.go

```diff
@@ -7,10 +7,10 @@ import (
 	"github.com/openshift/installer/pkg/types"
 )
 
-// knownRegions is a list of AWS regions that the installer recognizes.
-// This is subset of AWS regions and the regions where RHEL CoreOS images are published.
-// The result is a map of region identifier to region description
-func knownRegions(architecture types.Architecture) map[string]string {
+// knownPublicRegions is a subset of public AWS regions and regions where RHEL CoreOS images are published.
```

Comment:
I still think that you are using the term subset incorrectly here to mean intersection.
Either
```suggestion
// knownPublicRegions is the subset of public AWS regions where RHEL CoreOS images are published.
```
or
```suggestion
// knownPublicRegions is the intersection of public AWS regions and regions where RHEL CoreOS images are published.
```